shared/attributes.asciidoc

```diff
@@ -138,12 +138,13 @@ Elastic Cloud
 :ece:         Elastic Cloud Enterprise
 :eck:         Elastic Cloud on Kubernetes
 :cloud:       https://www.elastic.co/guide/en/cloud/current
-:ess-trial:   https://www.elastic.co/cloud/elasticsearch-service/signup?baymax=docs-body&elektra=docs
-:ess-product: https://www.elastic.co/cloud/elasticsearch-service?baymax=docs-body&elektra=docs
-:ess-console: https://cloud.elastic.co?baymax=docs-body&elektra=docs
+:ess-trial:   https://www.elastic.co/cloud/elasticsearch-service/signup{ess-utm-params}
+:ess-product: https://www.elastic.co/cloud/elasticsearch-service{ess-utm-params}
+:ess-console: https://cloud.elastic.co{ess-utm-params}
 :ess-console-name: {ess} Console
-:ess-deployments: https://cloud.elastic.co/deployments?baymax=docs-body&elektra=docs
-:ess-baymax:  ?baymax=docs-body&elektra=docs
+:ess-deployments: https://cloud.elastic.co/deployments{ess-utm-params}
+:ess-baymax:  {ess-utm-params}
+:ess-utm-params: ?page=docs&placement=docs-body
```

Comment:
Variables have to be set prior to their use. In this layout, you attempt to use the variables before they are set, leading to bad links throughout the docs.

Reply:
To clarify: You are trying to use the `{ess-utm-params}` var on L141-L143, but it isn't defined and set until L147.  `:ess-utm-params:` must be defined before it is used for it to be interpreted.

Reply:
An example can be seen on [this](https://docs_2737.docs-preview.app.elstc.co/guide/en/observability/current/ingest-traces.html) page (ctrl + f "Try it out for free").

Reply:
Thanks — Very good catch. I'll push a commit to reorder the attributes and check that this is fixed.